client/components/gridicon/index.jsx

```diff
@@ -16,7 +16,7 @@ var React = require( 'react' ),
 
 require( './style.scss' );
 
-var Gridicon = React.createClass( {
```

Comment:
Does this component need to exist? Can we convert projects using it to use the [`gridicons` package](https://www.npmjs.com/package/gridicons) instead?